src/TimeSlotGroup.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes, Component } from 'react'
+import TimeSlot from './TimeSlot.jsx'
+import date from './utils/dates.js'
+import localizer from './localizer'
+
+export default class TimeSlotGroup extends Component {
+  static propTypes = {
+    timeslots: PropTypes.number.isRequired,
+    step: PropTypes.number.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showLabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timeGutterFormat: PropTypes.string,
+    culture: PropTypes.string
+  }
+  static defaultProps = {
+    timeslots: 2,
+    step: 30,
+    isNow: false,
+    showLabels: true
+  }
+
+  renderSlice(i, content, value) {
```

Comment:
`i` is a bit unclear here for me, can we change it to `isFirst` or something and coerce the `0` to `true in `renderSlices`